scripts/build/testing.mk

```diff
@@ -24,8 +24,8 @@ test-all: test-unit test-integration test-ledger-mock test-race
 .PHONY: test-system
 test-system: build
 	mkdir -p ./tests/systemtests/binaries/
-	cp $(BUILDDIR)/simd$(if $(findstring v2,$(COSMOS_BUILD_OPTIONS)),v2) ./tests/systemtests/binaries/
-	$(MAKE) -C tests/systemtests test
+	cp $(BUILDDIR)/simdv2 ./tests/systemtests/binaries/
+	COSMOS_BUILD_OPTIONS=v2 $(MAKE) -C tests/systemtests test
```

Comment:
in theory this prohibits to run systemtests that expected something extra set in `COSMOS_BUILD_OPTIONS`. So we probably do not need that, given that it is already appended when in the makefile

Reply:
that value is not exported back to the shell so the systems test will not use it.  It's just a `make` variable.  If you remove the build options in this make target, the job will fail again because it is looking for `simd` and not `simdv2`